modules/dnn/src/layers/mvn_layer.cpp

```diff
@@ -289,10 +288,28 @@ class MVNLayerImpl CV_FINAL : public MVNLayer
             {
                 Mat inpRow = inpMat.row(i);
                 Mat outRow = outMat.row(i);
-
+                float weight = 1.f;
+                float bias = 0.f;
+                if (fuse_batch_norm)
```

Comment:
We need to make it true at
https://github.com/opencv/opencv/blob/3dd916882a3001b4e65bc3173e23f0e97fe3b434/modules/dnn/src/layers/mvn_layer.cpp#L74-L83